resolver/init.sls

```diff
@@ -2,7 +2,13 @@
 ##### Salt Formula For Resolver #####
 #####################################
 
+{% from "resolver/map.jinja" import resolver with context %}
+
 # Resolver Configuration
+resolver:
+  pkg.removed:
+    - name: {{resolver.package}}
```

Comment:
You might want to adjust whitespace here to `{{ resolver.package }}` as in the other cases.